### DISCUSSION TITLE
Divi Builder and qtranslate

### DISCUSSION THREAD

**Discussion by zbubicic** [qtranslate/qtranslate-xt#1185] [Q&A]
cant use qtransalate in Divi Builder.
Any help?
Thanx

**Comment by mtonumaa** [comment]
qTranslate XT (this page) works totally fine with Divi
For instance, qTranslate XT 3.15.2 works perfect with Divi 4.23.4.

Install is super easy. 
1) goto https://github.com/qtranslate/qtranslate-xt
2) Download zip from the green "Code" button
3) Instructions are on the same page, just scroll down. 

Do not use qTranslate X (the old one), it is not maintained or updated for years now.
